tools/rate-limit-test/rate-limit-test.py

```diff
@@ -0,0 +1,42 @@
+import argparse
+import logging
+import requests 
+import time
+import uuid
+import sys
+
+# Allow users to modify a few portions of the script.
+parser = argparse.ArgumentParser(description='A simple script to test rate limiting on a Fleet instance via brute forcing my device pages. Each GET request will contain a new uuid. Rate limiting will start at 720 requests sent in a period of 60 minutes.')
+parser.add_argument('--ip', help='Supply the ip of your Fleet instance, default set to https://localhost:8080', default='https://localhost:8080')
+parser.add_argument('--requests', help='Specify the number of GET requests sent to the target, default is set to 900', default=900)
+#parser.add_argument('--logfile', help='Name of logfile for output', default='rate_limit_output.log')
+args = parser.parse_args()
+
+# Suppress some annoying text that warns of no ssl cert being present.
+requests.packages.urllib3.disable_warnings(requests.packages.urllib3.exceptions.InsecureRequestWarning)
+
+# Header values that need to be set for this test
+headers = {'X-Forwarded-For': '127.0.0.1'}
```

Comment:
Let's add a comment that this header has to be set by a trusted proxy in order for rate limiting to work.